src/index.ts

```diff
@@ -40,6 +41,7 @@ export = {
         '@typescript-eslint/no-unused-vars': ['error', { args: 'none' }],
         '@typescript-eslint/no-use-before-define': ['error', { functions: false, classes: false, variables: false, typedefs: false }],
         '@typescript-eslint/no-useless-constructor': 'error',
+        '@typescript-eslint/quotes': ['error', 'single', { avoidEscape: true, allowTemplateLiterals: false }],
```

Comment:
New rule in typescript-eslint:
https://github.com/typescript-eslint/typescript-eslint/pull/762

Same as:
https://github.com/standard/eslint-config-standard/blob/eb187deec36e72bfea4034db1a81b0d7414f814a/eslintrc.json#L179